spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording.yml

```diff
@@ -5253,6 +5253,12 @@ https://localhost:8443/ovirt-engine/api/vms/3a9401a0-bf3d-4496-8acf-edd3e903511f
                     </topology>
                 </cpu>
                 <cpu_shares>0</cpu_shares>
+            <custom_properties>
```

Comment:
Please align it with existing content

Reply:
sure, actually it does not work for some reason, I will probably delete this part.